doozer/doozerlib/backend/rebaser.py

```diff
@@ -1166,7 +1166,7 @@ def _add_build_repos(self, dfp: DockerfileParser, metadata: ImageMetadata, dest_
             konflux_lines += [
                 "USER 0",
                 "RUN mkdir -p /tmp/art/yum_temp; mv /etc/yum.repos.d/*.repo /tmp/art/yum_temp/ || true",
-                f"COPY .oit/{self.repo_type}.repo /etc/yum.repos.d/",
+                f"COPY .oit/art-{self.repo_type}.repo /etc/yum.repos.d/",
```

Comment:
@vfreex We are writing files to `.oit/{self.repo_type}.repo`, in a different part of the codebase right? Dont that need to be updated as well?

Reply:
Makes sense. I fixed that part and will test it in ocp4-konflux job.